src/Psr/CacheItemPool/CacheItemPoolDecorator.php

```diff
@@ -149,7 +148,7 @@ public function getItems(array $keys = [])
     /**
      * {@inheritdoc}
      */
-    public function hasItem($key)
+    public function hasItem($key): bool
```

Comment:
Can we also add argument types so we are aligned with the interface?

Reply:
added (needed to drop "psr/cache": "^1.0  for it to work)